### PR TITLE
Created New Owner Inquiry Page

### PR DESCRIPTION
**Pull request by rohitkphilip**

### CODE REVIEW

File: package.json
Comment (adityabaravkar) [inline package.json:1]: I think React picks up properties in .env without the need for dotenv.

File: src/components/Contracts/OwnerManagerContracts/NewOwnerInquiry.jsx [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+        console.log("New Owner Inquiry UseEffect");
+        
+        const fetchData = async () => {
+        setShowSpinner(true);
+        
+        // const response = await fetch(`https://l0h6a9zi1e.execute-api.us-west-1.amazonaws.com/dev/properties/110-000096`)
+        
+        // rohit
+        // const response = await fetch(`https://l0h6a9zi1e.execute-api.us-west-1.amazonaws.com/dev/properties/${profileId}`)
+        const response = await fetch(`https://l0h6a9zi1e.execute-api.us-west-1.amazonaws.com/dev/properties/600-000003`)
```
Comment (adityabaravkar) [inline src/components/Contracts/OwnerManagerContracts/NewOwnerInquiry.jsx:35]: Dynamic profile UID

**Review by adityabaravkar — CHANGES_REQUESTED** [review]

File: package.json
Comment (rohitkphilip) [reply package.json:1]: Ok. I removed it.

File: package-lock.json
Comment (adityabaravkar) [inline package-lock.json:1]: Can you undo the changes of package-lock.json?

**Pull request closed without merging**